book/website/reproducible-research/licensing/licensing-ml.md

```diff
@@ -0,0 +1,65 @@
+(rr-licensing-ml)=
+# Machine Learning Model Licenses
+
+Like a software license, a Machine Learning (ML) model license governs the use, redistribution of the model and/or algorithm, and distribution of any derivatives of it. However, there are other components to an AI system, such as [data](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-data.html), 
+[source code](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-software.html), or applications, which may have their own separate licenses. ML model licenses may restrict the use of the model for specific scenarios for which, due to ethics-informed concerns and/or technical limitations of the model informed by its model card, the licensor is not comfortable that the model is used.
+
+While many ML models may utilise open software licensing (e.g. MIT, Apache 2.0), 
+there are a number of ML model-specific licenses that may be developed for a specific model (e.g. [OPT-175B license](https://github.com/facebookresearch/metaseq/blob/main/projects/OPT/MODEL_LICENSE.md), [BigScience BLOOM RAIL v1.0 License](https://https://bigscience.huggingface.co/blog/the-bigscience-rail-license)), 
+company (e.g. [Microsoft Data Use Agreement for Open AI Model Development](https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RE4Rjfq)), or series of models (e.g. [BigScience OpenRAIL-M](https://www.licenses.ai/blog/2022/8/26/bigscience-open-rail-m-license) (Responsible AI License)). 
+
+In summary, the growing list of ML licenses reflects the understanding that the ML model and other artefacts are distinct from the code, and thus in need of new licensing options. 
```

Comment:
It might be worth adding a sentence here about artifacts? I know it's a commonly used term in the field, but would be great to have a short description here for folks who might not know, but are on the page. This could also be a glossary addition, which could be linked here: https://the-turing-way.netlify.app/afterword/glossary.html
```suggestion
In summary, the growing list of ML licenses reflects the understanding that the ML model and other artifacts are distinct from the code, and thus in need of new licensing options. 
```

Reply:
In terms of spelling: I'm seeing both artifacts and artefacts online – a distinction between British and American English, I know! Ignore the suggestion spelling changes above, but the comment above applies.